### PR TITLE
test: e2e for upper navigation menu

### PR DESCRIPTION
**Pull request by abilevych**

### CODE REVIEW

File: tests/e2e/features/artifacts.feature [hunk 1]
```diff
@@ -269,4 +269,67 @@ Feature: Artifacts - UI
...
+    Then Element with "href and text" "'https://zksync.io/ethos' and 'Freedom is our mission'" should be "clickable"
+    Then Element with "href and text" "'https://zksync.io/hyperscalability' and 'Hyperscalibility'" should be "visible"
+    Then Element with "href and text" "'https://zksync.io/hyperscalability' and 'Hyperscalibility'" should be "clickable"
+    Then Element with "href and text" "'https://zksync.io/security' and 'Security'" should be "visible"
+    Then Element with "href and text" "'https://zksync.io/security' and 'Security'" should be "clickable"
+    Then Element with "href and text" "'https://zksync.io/ux' and 'Game-changing UX'" should be "visible"
+    Then Element with "href and text" "'https://zksync.io/ux' and 'Game-changing UX'" should be "clickable"
+
+  @id1531 @upperNavigationMenu
+  Scenario: Check artifacts on the Upper navigation menu (Network)
```
Comment (pcheremu) [inline tests/e2e/features/artifacts.feature:317]: based on id1530 here we can also add: 
Then Element with "partial text" "<Text>" should be "visible"

ZKSYNC ERA (V2)
ZKSYNC LITE (V1)
ECOSYSTEM

**Review by pcheremu — APPROVED** [review]

**Review by pcheremu — DISMISSED** [review]
Everything looks good.

**Comment by pcheremu** [comment]
@abilevych please keep in mind, in depends of who will be merged first, PR #55 will call merge conflict

**Review by JackHamer09 — DISMISSED** [review]

**Review by pcheremu — APPROVED** [review]

**Review by obmatter — DISMISSED** [review]
Reviewed - approved.

**Review by JackHamer09 — APPROVED** [review]

**Pull request closed without merging**